src/Microsoft.CodeAnalysis.Analyzers/Core/MetaAnalyzers/PreferIsKindAnalyzer.cs

```diff
@@ -36,6 +36,8 @@ public override void Initialize(AnalysisContext context)
 
             context.RegisterCompilationStartAction(context =>
             {
+                var nullableOfT = context.Compilation.GetOrCreateTypeByMetadataName(WellKnownTypeNames.SystemNullable1);
```

Comment:
💡 Use `GetSpecialType`?

```suggestion
                var nullableOfT = context.Compilation.GetSpecialType(SpecialType.System_Nullable_T);
```

Reply:
@sharwell Probably not even needed? Could check directly for:

```csharp
conditionalAccess.Operation.Type.OriginalDefinition.SpecialType == SpecialType.System_Nullable_T
```

instead of retrieving an INamedTypeSymbol and using SymbolEqualityComparer.

Reply:
Nice 👍